.changeset/nasty-starfishes-provide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hashicorp/react-button': patch
+---
+
+Fixes contrast issue on brand background by using proper brand values.
```

Comment:
Here's a before after from parallel changes in [react-components-storybook](https://react-components-storybook.vercel.app):

## Before

<img width="763" alt="before" src="https://user-images.githubusercontent.com/4624598/137326374-8f1ca9b0-3a26-4931-9fa2-76fbfa633bcc.png">

## After

<img width="791" alt="after" src="https://user-images.githubusercontent.com/4624598/137326463-aa6a97df-e4ff-485a-a838-c5f84bbd36b1.png">